examples/fluid/hydrostatic_water_column_2d.jl

```diff
@@ -31,7 +31,8 @@ tank = RectangularTank(fluid_particle_spacing, initial_fluid_size, tank_size, fl
 smoothing_length = 1.2 * fluid_particle_spacing
 smoothing_kernel = SchoenbergCubicSplineKernel{2}()
 
-viscosity = ArtificialViscosityMonaghan(alpha=0.02, beta=0.0)
+alpha = 0.02
+viscosity = ArtificialViscosityMonaghan(alpha=alpha, beta=0.0)
```

Comment:
Why?

Reply:
Because alpha is used in a trixi_include.